ansible/roles_ocp_workloads/ocp4_workload_opendatahub_old/templates/open-data-hub/subscription.yml.j2

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: opendatahub-operator
+  namespace: {{ namespace }}
+spec:
+  channel: legacy
+  installPlanApproval: Automatic
+  name: opendatahub-operator
+  source: community-operators
```

Comment:
Didn’t you want to add a snapshot?  Which means you will also need a Catalogsource and operator group.